src/components/Kurlsh.js

```diff
@@ -836,6 +847,31 @@ class Kurlsh extends React.Component {
     }
   }
 
+  // add versions like "1.19.x" to the list of installable versions
+  addK8sVersions(actualVersions) {
```

Comment:
i'll prefer this function to be arrow function.
also you can change code inside if you want to follow that pattern too:

```suggestion
  addK8sVersions = (actualVersions) => {
    // get a list of the distinct minor versions
    const minorVersionsRegex = /^1\.[0-9]+/g;
    let minorVersions = []

    actualVersions.forEach(v => {
      const matches = v.version.match(minorVersionsRegex);
      if (matches && matches.length === 1 && !minorVersions.includes(matches[0])) {
        minorVersions.push(matches[0])
      }
    })

    // for each minor version, find the first version in the actualVersions array that matches
    // and insert `1.minor.x` before it
    minorVersions.forEach(mv => {
      const isMatch = actualVersions.find(av => av.version.startsWith(mv));
      if (!!isMatch) {
        actualVersions.splice(actualVersions.indexOf(isMatch), 0, {version: mv+".x"});
      }
    })

    return actualVersions
  }
```